stock_picking_product_barcode_report/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    barcode_default_format = fields.Selection(
```

Comment:
@CarlosRoca13 @pedrobaeza Shouldn't this defined in a more generic module ?

Reply:
So, maybe the field name too generic is.

Reply:
Very Yoda your sentence is, hehe. @CarlosRoca13 please change it to `barcode_report_default_format`.